usertable.cpp

```diff
@@ -432,10 +432,9 @@ void UserTable::OnEvent(InotifyEvent& rEvt)
           oldpos = pos + 2;
         }
         else if (cs[px] == '&') {     // numeric mask
-          char* s;
-          asprintf(&s, "%u", (unsigned) rEvt.GetMask());
+          char s[11];
```

Comment:
The size of the buffer was chosen because of the output format `%u` and not the possible values of the mask.
A 32 bit unsigned int uses at most 10 characters (4,294,967,295), so 11 to account for the terminating null byte.

I think `new` throws an exception when the allocation fails. Anyway, this method avoids extra checks, as it uses stack memory for the temporary buffer and snprintf won't truncate the output since the buffer as enough space for the unsigned representation and the terminating null byte.